packages/@aws-cdk/cloudformation-diff/lib/diff/util.ts

```diff
@@ -11,6 +11,7 @@
  */
 export function deepEqual(lvalue: any, rvalue: any): boolean {
   if (lvalue === rvalue) { return true; }
+  if (parseFloat(lvalue) === parseFloat(rvalue)) { return true; }
```

Comment:
I had to double check some edge cases here, but it seems this is fine to do.

For the benefit of future scholars, can you add a comment line explaining why we're doing this?